### ISSUE TITLE
No way to sideload/test iPad Outlook add-ins, and sideload instructions missing Outlook information

### ISSUE THREAD

**Issue by gitcnd** [OfficeDev/office-js-docs-pr#2947]
The doc ( https://docs.microsoft.com/en-us/office/dev/add-ins/testing/sideload-an-office-add-in-on-ipad-and-mac#see-also ) does not include any instructions for testing Outlook add-ins; only for Excel/Word (iOS) and Macbook (missing iOS).

I can see many add-in's available on my iPad (e.g. Evernote), but there is no "Developer" or other "UI" element like the Excel/Word instructions suggest should be there.

Please fix your DOCUMENTATION to tell us how to test our Add-ins.

## Your Environment
* Platform iPad
* Host Outlook
* Office version number: current
* Operating System: current
 
## Expected behavior
Support for developers

## Current behavior
No support, no documentation for Outlook developers.

## Steps to reproduce
1. try ti side-load any Outlook add-in on an iPad.

## Context
I need to test before I submit to AppSource !

**Comment by ElizabethSamuel-MSFT** [comment]
@gitcnd We'll work on clarifying the documentation. In the meantime, the way to test your Outlook add-in on mobile should be something like the following.

1. [Sideload on the web, Windows, or Mac](https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing). The add-in should then show up in all endpoints declared in the manifest. Right now, I believe we don't have an option for you to directly sideload to a mobile device.
2. Guidance for [testing on mobile](https://docs.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-mobile-addins#testing-your-add-ins-on-mobile).

Hope this unblocks you.

Thanks.

**Comment by gitcnd** [comment]
@ ElizabethSamuel-MSFT - thanks for the hint, however, it does not work for iOS.

I did as you suggested, and my add-in shows up and works fine in the Android app, but it does not show up at all on my iPhone.

Other store apps I added do show up and work on both the iPhone and Android (e.g. MojoLaLa), but my side-loaded one does not show at all on my iPhone.

Note also: I am unable to test on iPad because of this blocking set of bugs: https://github.com/OfficeDev/office-js/issues/2139
